build/makelib/common.mk

```diff
@@ -22,6 +22,7 @@ PLATFORM := $(GOOS)_$(GOARCH)
 else
 GOOS := $(word 1, $(subst _, ,$(PLATFORM)))
 GOARCH := $(word 2, $(subst _, ,$(PLATFORM)))
+export GOOS GOARCH
```

Comment:
I dont understand why we need to explicitly call `export`? Doesnt the two lines above do the same? Or does that just set the variable at a local scope?